about.md

```diff
@@ -18,23 +18,54 @@ They bring together design, technical, and policy components to give a view of w
 
 Data patterns can help us build trust with users by ensuring that our digital services are private, safe, and equitable by design.  
 
-<em>(more content coming)</em>
+For **users** (people using our services), being able to understand how our services work and use data is essential to making informed decisions for themselves or the people they care for.
 
-### Dark patterns and Good patterns
+For **teams** (people building our services), being able to responsibly reuse and adapt components makes it easier to build the best services we can - ones that consistently follow rules or best practices and work with one another in order to meet users' needs.
+
+### Avoiding dark patterns
 
 Working with users to develop trusted data patterns can also help us avoid dark patterns.
 
-[Dark patterns](https://www.darkpatterns.org/) are "tricks used in websites and apps that make you do things that you didn't mean to, like buying or signing up for something". They [use the power of design](https://dapde.de/en/) to benefit a product's owners or creators more than the people who use it.
+[Dark patterns](https://www.darkpatterns.org/) are "tricks used in websites and apps that make you do things that you didn't mean to, like buying or signing up for something".
+They [use the power of design](https://dapde.de/en/) to benefit a product's owners or creators more than the people who use it.
 
 Dark patterns can do this in many different ways, but mainly by limiting:
 
 * **Consent** - the ability for people to choose what data they do and don't want to have collected, used, stored or shared
-* **Openness** - the ability for people to see and understand how a service was designed and how it works
+* **Openness** - the ability for people to see, understand, and participate in how a service is designed and how it works
 * **Accountability** - the ability to give, receive, and respond to feedback
 
-For example,
+#### Example - Cookie pop-up
+
+For example, we're required by law to ask people for their consent when we want to collect data through cookies.
```

Comment:
I'm pretty sure this is only true in the UK. I don't think Canada has a law on this. Which makes me think, if I'm wrong about it, it would be great to link to the law.